R/query.R

```diff
@@ -46,12 +49,19 @@
 #' docdb_query(src, "iris", query = "1.5")
 #' docdb_query(src, "iris", query = "Petal.Width:1.5")
 #'
-#' # Mongo
+#' # MongoDB
 #' src <- src_mongo()
 #' if (docdb_exists(src, "mtcars")) docdb_delete(src, "mtcars")
 #' docdb_create(src, "mtcars", mtcars)
 #' docdb_query(src, query = '{"mpg":21}')
 #' docdb_query(src, query = '{"mpg":21}', fields = '{"mpg":1, "cyl":1}')
+#' 
+#' # SQLite
+#' src <- src_sqlite()
+#' if (docdb_exists(src, "mtcars")) docdb_delete(src, "mtcars")
+#' docdb_create(src, "mtcars", mtcars)
+#' docdb_query(src, fields = c("mpg", "cyl"))
```

Comment:
i think maybe you weren't done with the query method, but .... the query methods are missing the `key` param input; i'd prefer a data.frame to be returned, not a data.table object